hyperSpec/vignettes/fileio.Rmd

```diff
@@ -1288,14 +1289,14 @@ Thus information like the excitation wavelength is lost.
 \index{Nicolet!Map}
 \index{Nicolet!Infrared}
 
-The function `read.ENVI.Nicolet()`{.r} is a good example for a more specific import filter derived from a general filter for the respective file type.
+The function `read.ENVI.Nicolet()`{.r} is an excellent example of a more specific import filter derived from a generic filter for the particular file type.
 Nicolet FT-IR Imaging software saves some non-standard keywords in the header file of the ENVI data.
-These information can be used to reconstruct the $x$ and $y$ axes of the images.
+This information can be used to reconstruct the $x$ and $y$ axes of the images.
 The units of the spectra are saved as well.
 
 Function `read.ENVI.Nicolet()`{.r} thus first adjusts the parameters for `read.ENVI()`{.r}.
 Then `read.ENVI()`{.r} does the main work of importing the file.
-The resulting `hyperSpec`{.r} object is post-processed according to the special header entries.
+The resulting `hyperSpec`{.r} object is post-processed according to the unique header entries.
 
```

Comment:
I'm not sure if "unique" is appropriate here. I didn't read the context, but if I understand correctly, this means "a certain type of headers that trigger something"